dtfs-central-api/tsconfig.json

```diff
@@ -24,9 +24,9 @@
       // "useDefineForClassFields": true,                  /* Emit ECMAScript-standard-compliant class fields. */
 
       /* Modules */
-      "module": "CommonJS",                             /* Specify what module code is generated. */
+      "module": "node16",                             /* Specify what module code is generated. */
       "rootDir": "./",                                  /* Specify the root folder within your source files. */
-      "moduleResolution": "node",                       /* Specify how TypeScript looks up a file from a given module specifier. */
+      "moduleResolution": "node16",                       /* Specify how TypeScript looks up a file from a given module specifier. */
```

Comment:
This change was required to allow us to recognise the multi-entry point functionality declared by
```
"exports": {
  ".": "./src/index.ts",
  "./sql-db-connection": "./src/sql-db-connection/index.ts"
},
```
in the `libs/common/package.json` so we can use the following import here in the `dtfs-central-api`
```
import { SqlDbDataSource } from '@ukef/dtfs2-common/sql-db-connection';
```